test/framework/src/main/java/org/elasticsearch/test/rest/ESRestTestCase.java

```diff
@@ -642,9 +642,13 @@ private void wipeCluster() throws Exception {
 
     protected static void wipeAllIndices() throws IOException {
         boolean includeHidden = minimumNodeVersion().onOrAfter(Version.V_7_7_0);
+        boolean includeSystem = minimumNodeVersion().onOrAfter(Version.V_8_0_0);
```

Comment:
Do you want to have a constant version somewhere for when the `allow_system_index_access` flag was added?